scripts/releases/upload-release-assets-for-dotslash.js

```diff
@@ -0,0 +1,233 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+const {REPO_ROOT} = require('../shared/consts');
+const {getWithCurl} = require('./utils/curl-utils');
+const {
+  processDotSlashFileInPlace,
+  validateDotSlashArtifactData,
+} = require('./utils/dotslash-utils');
+const {
+  FIRST_PARTY_DOTSLASH_FILES,
+} = require('./write-dotslash-release-asset-urls');
+// $FlowFixMe[untyped-import] TODO: add types for @octokit/rest
+const {Octokit} = require('@octokit/rest');
```

Comment:
nit: we usually do rest calls ourselves manually, to avoid importing the oktokit libraries, but we usually don't have to handle files... if the logic is more complex, it might make sense to import them.

Reply:
Resolving per https://github.com/facebook/react-native/pull/52930/files/b6837afa477a2203d76891599224083b81ef0b8a#r2267078651.